wandb/integration/yolov8/yolov8.py

```diff
@@ -198,9 +245,80 @@ def callbacks(
             "on_train_end": self.on_train_end,
             "on_model_save": self.on_model_save,
             "teardown": self.teardown,
+            "on_val_start": self.on_val_start,
+            "on_val_end": self.on_val_end,
         }
 
 
+def add_images_to_validation_table(
+    logger: WandbCallback,
+    validator: BaseValidator,
+    images: List[Tuple[wandb.Image, wandb.Image]],
+    scores: List[Dict[str, float]],
+    batch_id: int,
+) -> None:
+    """Utility to add a wandb.Image, predictions and scores to the validation table"""
+    objects = sorted(validator.names.values())
```

Comment:
there seems to be an implicit ordering of the order of class by doing this sorting. It doesn't seem very readable (given that it happens in 2 different functions). I would suggest refactoring this logic, so we have the table columns and values updated in the same place.